opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/jms/JmsTracingTest.java

```diff
@@ -0,0 +1,48 @@
+package io.opentracing.contrib.spring.cloud.jms;
+
+import static org.awaitility.Awaitility.await;
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+
+import io.opentracing.ActiveSpan;
+import io.opentracing.contrib.spring.cloud.MockTracingConfiguration;
+import io.opentracing.mock.MockSpan;
+import io.opentracing.mock.MockTracer;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jms.core.JmsTemplate;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+/**
+ * @author <a href="mailto:ales.justin@jboss.org">Ales Justin</a>
+ */
+@SpringBootTest(
+  classes = {MockTracingConfiguration.class, JmsTestConfiguration.class})
+@RunWith(SpringJUnit4ClassRunner.class)
+public class JmsTracingTest {
```

Comment:
I would like to see span wiring test when server span created in spring-web would be a parent of the span created in JMS. In other words you will call traced serer REST endpoint and inside there will be a JMS invocation.